cmd/rocsp-tool/client_test.go

```diff
@@ -50,6 +50,32 @@ func makeClient() (*rocsp.RWClient, clock.Clock) {
 	return rocsp.NewWritingClient(rdb, 500*time.Millisecond, clk, metrics.NoopRegisterer), clk
 }
 
+// TODO(#6517) remove this test.
```

Comment:
This is simply moved from the bottom half to the top now that we're calling it inside of the combined client test funcs.